changelog.txt

```diff
@@ -1,19 +1,36 @@
+4.23.0
+============
+
+New Features:
+**Windows Account Manager (WAM) is now available on .NET classic**. See issue [#2181](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2181) and [#2182](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2182) for details and https://aka.ms/msal-net-wam.
+**Proof of Possession (Signed HTTP Request) for confidential clients now support key management**. See issue [#2013](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2013) and ()[] for details.
```

Comment:
New static `OperatingSystemAccount` property on `PublicClientApplication`

Reply:
i inculuded more on this on line 5. thank you